app/src/main/java/gholzrib/arctouchchallenge/core/utils/CheckConnection.java

```diff
@@ -0,0 +1,42 @@
+package gholzrib.arctouchchallenge.core.utils;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.location.LocationManager;
+import android.net.ConnectivityManager;
+import android.widget.Toast;
+
+
+public class CheckConnection {
+
+    private static final String CONNECTION_ERROR_MESSAGE = "No internet connection, please try again later";
```

Comment:
This class is part of a group of utility classes that I have been gathering through the years. It usually places the text into the string.xml, but to make the creation of the class easier I decided to use it as an String.